webstruct/features/global_features.py

```diff
@@ -99,4 +97,5 @@ def _add_pattern_features(feature_dicts, pattern, out_value, missing_value, sepa
 
         # FIXME: there should be a cleaner/faster way
         if not all(v == out_value for v in values):
+            values = [str(v) for v in values]
```

Comment:
This is not correct in Python 2, as you'll be casting unicode features to str (i.e. to bytes).